documentation/book/proc-restricting-access-to-listeners-using-network-policies.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// assembly-configuring-kafka-listeners.adoc
+
+[id='proc-restricting-access-to-listeners-using-network-policies-{context}']
+= Restricting access to Kafka listeners using NetworkPolicies
+
+.Prerequisites
+
+* An {ProductPlatformName} cluster with SDN supporting Ingress NetworkPolicies
+* A running Cluster Operator
+
+.Procedure
+
```

Comment:
_Suggest breaking up the steps:_

1. Deploy a Kafka cluster.

2. Open the `Kafka.spec.kafka` resource.

3. In the `networkPolicyPeers` field, define the application pods or namespaces that will be allowed to access the Kafka cluster.

  - For example, to configure a `tls` listener to allow connections only from application pods with the label `app` set to `kafka-client`:

Reply:
These are not necessarily separate steps. You do not need to deploy the cluster and then edit the network policies. If your cluster is not running yet, you should edit the YAML file first and then it is deployed with the command from the next step. Maybe we should just remove the _Deploy Kafka cluster and_ part?

Reply:
Thanks for the explanation. Yes, let's remove step 1 (deploy a Kafka cluster). As long as it's clear which resource the user needs to edit.